vuepress/docs/next/tutorials/ecr/publish-simple-bundle.md

```diff
@@ -0,0 +1,130 @@
+---
+sidebarDepth: 2
+redirectFrom: /next/tutorials/ecr/tutorials/create-ecr-bundle-from-git.html
+---
+# Build and Publish a Simple Bundle
+
+## Overview
+This tutorial shows you how to create a simple Entando bundle and deploy it into the Entando Component Repository. This involves manually defining a bundle with a single widget, checking your bundle artifacts into git, apply the Entando bundle custom resource to Kubernetes, and then installing the bundle into your application. 
+
+## Prerequisites
+* Use the [Entando CLI](../../docs/reference/entando-cli.md#check-environment) tool to verify you have the necessary prerequisites in place (e.g. Java, npm, git, etc.). You will also need your git credentials, an available git repository, and an Entando instance. Some of the following steps can be simplified by using the `ent prj` command and its publication system (pbs) convenience methods. The manual steps are also provided in those cases.
+
+## Create the project folder 
+We'll create a parent folder and then a child bundle folder. In a project generated by the Entando Component Generator the microservice and micro frontend source files also live under the parent folder.
+
+``` sh
+mkdir -p example-bundle/bundle; cd example-bundle/bundle 
+```
+## Add a simple widget
+
+Create a widget directory and descriptor file:
+``` sh
+mkdir widgets
+nano widgets/example-widget.yaml
```

Comment:
You get to field the nano related support requests ;)

On a serious note should we just recommend creating the file?

mkdir widgets
In the widgets directory create a file called example-widget.yaml ....

Reply:
Yeah, I'm convinced after further thought. It was vim before I touched this doc again so I was looking for what would work better. Leaving it up to the dev is probably better/safer/less fragile.

Reply:
Done